android/src/main/java/com/swmansion/gesturehandler/core/TapGestureHandler.kt

```diff
@@ -152,6 +152,24 @@ class TapGestureHandler : GestureHandler<TapGestureHandler>() {
     end()
   }
 
+  override fun hasCommonPointers(other: GestureHandler<*>): Boolean {
+    if (super.hasCommonPointers(other)) {
+      return true
+    }
+
+    val cachedPointersIDs = getCachedPointersIDs()
+    val otherCachedPointerIDs = other.getCachedPointersIDs()
+    val nextCachedPointerID = getNextCachedPointerID()
+
+    for (i in 0 until nextCachedPointerID) {
```

Comment:
Is this necessary? If I understand correctly, pointers will be clustered at the beginning anyway, so we can iterate over entire array and break when encountering `-1` no?

If so, then getter for getNextCachedPointerID can be removed.